From 54d029c8dc41da98375251f38980909b9442fa46 Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Tue, 23 Aug 2011 17:54:34 +0800
Subject: [PATCH 336/678] ENGR00155240 mx53_smd: Fix a bug that hangs on
 printf() in board_init

remove printf() because serial interface is not ready in board_init()

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 board/freescale/mx53_smd/mx53_smd.c |    6 +-----
 1 files changed, 1 insertions(+), 5 deletions(-)

diff --git a/board/freescale/mx53_smd/mx53_smd.c b/board/freescale/mx53_smd/mx53_smd.c
index b9bd250..eda4eec 100644
--- a/board/freescale/mx53_smd/mx53_smd.c
+++ b/board/freescale/mx53_smd/mx53_smd.c
@@ -414,12 +414,8 @@ void setup_pmic_voltages(void)
 	/* increase VDDGP as 1.25V for 1GHZ */
 	value = 0x5e;
 	do {
-		if (0 != i2c_write(0x48, 0x2e, 1, &value, 1)) {
-			printf("da9052_i2c_is_connected - i2c write failed.....\n");
-		} else {
-			printf("da9052_i2c_is_connected - i2c write success....\n");
+		if (0 == i2c_write(0x48, 0x2e, 1, &value, 1))
 			ret = 0;
-		}
 	} while (ret != 0 && retries--);
 	i2c_read(0x48, 60, 1, &value, 1);
 	value |= 0x1;
-- 
1.7.7.4

